integration-tests/manual-tests.md

```diff
@@ -241,3 +241,22 @@ $ snap remove bluez
           Vendor Specific data: 0000
           TPM Version:         01010000
           Manufacturer Info:   57454300
+
+# Test fwupd interface with uefi-fw-tools
+
+1. Ensure your BIOS support UEFI firmware upgrading via UEFI capsule format
+2. Install the uefi-fw-tools snap from the store
+3. Ensure the 'fwupd' interface is connected
```

Comment:
I've remove all the plugs (network,network-bind) and added all the syscall needed in the code.